diff --git a/node_modules/pdf-parse/index.js b/node_modules/pdf-parse/index.js
index e9fc367..f55e70b 100644
--- a/node_modules/pdf-parse/index.js
+++ b/node_modules/pdf-parse/index.js
@@ -8,19 +8,19 @@ let isDebugMode = !module.parent;
 //process.env.AUTO_KENT_DEBUG
 
 
-//for testing purpose
-if (isDebugMode) {
+// //for testing purpose
+// if (isDebugMode) {
 
-    let PDF_FILE = './test/data/05-versions-space.pdf';
-    let dataBuffer = Fs.readFileSync(PDF_FILE);
-    Pdf(dataBuffer).then(function(data) {
-        Fs.writeFileSync(`${PDF_FILE}.txt`, data.text, {
-            encoding: 'utf8',
-            flag: 'w'
-        });
-        debugger;
-    }).catch(function(err) {
-        debugger;
-    });
+//     let PDF_FILE = './test/data/05-versions-space.pdf';
+//     let dataBuffer = Fs.readFileSync(PDF_FILE);
+//     Pdf(dataBuffer).then(function(data) {
+//         Fs.writeFileSync(`${PDF_FILE}.txt`, data.text, {
+//             encoding: 'utf8',
+//             flag: 'w'
+//         });
+//         debugger;
+//     }).catch(function(err) {
+//         debugger;
+//     });
 
-}
+// }
